coding/python_testing.rst

```diff
@@ -4,6 +4,156 @@ Python Unit Testing
 
 This page provides technical guidance to developers writing unit tests for DM's Python code base.
 See :doc:`unit_test_policy` for an overview of LSST Stack testing.
+If you have legacy DM :mod:`unittest` ``suite``-based code please refer to tech note `SQR-012`_ for porting instructions.
```

Comment:
A few more words about what this means could help ("suite-based" isn't very clear). I am pretty sure you mean our tests that create a test suite and then call it, but it may be best to say so or include a representative snippet (something that makes it easy to tell if a test file is "old style").